src/styles/chat.css

```
@@ -1139,3 +1145,249 @@ a:hover { opacity: 0.85; }
+  opacity: 0.75;
+}
+
+/* Visually hidden, still announced. */
+.chat-sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)